.github/scripts/codespell.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+OUTPUT=$1
+
+echo "Run codespell..."
+codespell --quiet-level=2 --skip "*.h,*.cmake,*.c,*.hpp,*.cpp,*.sh,*.py,test/supp/*.supp" -i 0 -L "ASSER,Tne,ba,BA" || true
```

Comment:
`|| true` - means that it will always succeed, so it will never find any issues, so it is useless ;-)
It should fail, when it finds errors.